lib/process-services-cloud/src/lib/process/start-process/services/start-process-cloud.service.spec.ts

```diff
@@ -129,34 +116,6 @@ describe('StartProcessCloudService', () => {
             );
     });
 
-    it('should be able to start a created new process instance', (done) => {
```

Comment:
Please remove also the method that creates the process instance as it shoulnd't be used anymore

Reply:
done